models/public/forward_tacotron/description/forward_to_onnx.py

```diff
@@ -0,0 +1,206 @@
+import os
```

Comment:
This shouldn't be here, but in the parent directory. Also, please add a shebang line and make the file executable.